uint16 steer
uint16 throttle
uint16 delimiter
